.github/workflows/core-build-test.yml

```
@@ -0,0 +1,133 @@
+        if: steps.changes.outputs.code == 'true'
+        uses: actions-rs/cargo@v1
+        with:
+          command: test
+          args: --doc
+      - name: integration test dev environment
+        if: steps.changes.outputs.code == 'true'
+        env:
+          RUST_LOG: info
+          RUST_BACKTRACE: full
+          STARCOIN_WS: ${{ secrets.STARCOIN_WS }}
+        uses: actions-rs/cargo@v1
```

actionlint (1.7.7)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)